init/gstd.service.in

```diff
@@ -0,0 +1,11 @@
+[Unit]
+Description=Systemd service to launch GstD
+PartOf=graphical-session.target
+
+[Service]
+Type=forking
+ExecStart=@prefix@/bin/gstd -e
+ExecStop=@prefix@/bin/gstd -k
+
+[Install]
+WantedBy=graphical-session.target
```

Comment:
Can you explain this?

Reply:
We want to start gstd until all the graphical stuff is up, otherwise, the video-sinks won't appear because when gstd started, the environment didn't have some stuff defined yet.

I tried the After= directive but it didn't work, only the WantedBy= directive worked. The graphical-session.target is a systemd special target, check https://www.freedesktop.org/software/systemd/man/systemd.special.html